osu.Game.Tests/Visual/Gameplay/TestScenePracticeOverlay.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Overlays.Practice;
+using osu.Game.Rulesets;
+using osu.Game.Screens.Play;
+
+namespace osu.Game.Tests.Visual.Gameplay
+{
+    public class TestScenePracticeOverlay : TestSceneAllRulesetPlayers
+    {
+        [Cached]
+        private PracticePlayerLoader loader { get; set; } = new PracticePlayerLoader();
```

Comment:
how is this test even usable to test anything? i don't understand

Reply:
i need to work on it, originally there was no link for the overlay through song select so i was using this test for everything, i gutted it later on due to it causing me issues, i plan to make the full fledged test once i have the way to call the overlay better conceptualised